packages/theming/README.md

```diff
@@ -80,6 +80,35 @@ const LocalizedComponent = withTheme(StyledDiv);
 </ThemeProvider>;
 ```
 
+### Theme ids
+
+Each component has a `COMPONENT_ID` applied so you can target it in your own theme
+file to override the default look and feel. This table contains all the ids and which
+package they apply to.
+
+| Component     | COMPONENT_IDs                                                                                                                                                                                                                                                                                                                        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| avatars       | avatars.avatar                                                                                                                                                                                                                                                                                                                       |
+| buttons       | buttons.anchor, buttons.button, buttons.button&zwnj;\_group_view, buttons.icon, buttons.icon_button                                                                                                                                                                                                                                  |
```

Comment:
The `&zwnj;` entity is needed as styleguidist refused to let me just escape the underscores even though the underlying lib they use works fine with just escaping 🤷‍♂️